ops/ansible/roles/bfd-pipeline/templates/bfd-pipeline-service.sh.j2

```diff
@@ -5,8 +5,8 @@
 # Force UTC time
 export TZ=UTC
 
-export BFD_CCW_JOB_ENABLED={{ enabled if pipeline_instance_type == 'ccw' else 'false' }} {# derived from terraform control plane #}{{ '\n' }}
-export BFD_RDA_JOB_ENABLED={{ enabled if pipeline_instance_type == 'rda' else 'false' }} {# derived from terraform control plane #}{{ '\n' }}
+export BFD_CCW_JOB_ENABLED={{ job_enabled if pipeline_instance_type == 'ccw' else 'false' }} {# derived from terraform control plane #}{{ '\n' }}
+export BFD_RDA_JOB_ENABLED={{ job_enabled if pipeline_instance_type == 'rda' else 'false' }} {# derived from terraform control plane #}{{ '\n' }}
```

Comment:
This change, and the related changes in `ops/terraform/services/pipeline/user-data.sh.tftpl` to the `jq` queries, is necessary as the new SSM parameter `/bfd/${env}/pipeline/nonsensitive/ccw/slo/weekend_data_availability/verifier/enabled` would otherwise "shadow" the value of `/bfd/${env}/pipeline/nonsensitive/ccw/job/enabled` as the `last` in the existing `jq` queries would result in a duplicate `enabled` variable in the Ansible's "extra vars".

Changing the `jq` to instead include the _full_ name of the parameter (replacing `/` with `_`) to avoid this results in this variable coming `job_enabled` as its full name is `job/enabled`. I verified that there were no other variables changed that were relevant to the `bfd-pipeline` Ansible.

Reply:
See [c8c60fd](https://github.com/CMSgov/beneficiary-fhir-data/pull/2520/commits/c8c60fd8ac934f9bc994a040037925354cc8b2a3) for relevant commit.